test/Microsoft.ApplicationInsights.AspNetCore.Tests/RequestTrackingMiddlewareTest.cs

```diff
@@ -393,7 +397,7 @@ public void OnEndRequestFromDifferentInstrumentationKey()
             Assert.True(requestTelemetry.Duration.TotalMilliseconds >= 0);
             Assert.True(requestTelemetry.Success);
             Assert.Equal(CommonMocks.InstrumentationKey, requestTelemetry.Context.InstrumentationKey);
-            Assert.Equal("", requestTelemetry.Source);            
+            Assert.Equal("DIFFERENT_INSTRUMENTATION_KEY_HASH", requestTelemetry.Source);            
```

Comment:
this guy was validating the request telemetry without OperationCorrelationTelemetryIntitializer that set Source. Not sure what it was testing though :)